bot.js

```diff
@@ -1,197 +1,91 @@
 require("dotenv").config();
 
 let TelegramBot = require("node-telegram-bot-api");
+let game = require("./game.js");
 
 if (!process.env.TELEGRAM_TOKEN) {
   throw new Error("TELEGRAM_TOKEN env variable is missing");
 }
 
 const bot = new TelegramBot(process.env.TELEGRAM_TOKEN, { polling: true });
-let gameStart = false;
-let spentCities = new Set();
-let lastLetter = "";
 
-bot.on("polling_error", m => console.log(m));
+let session = [];
+let usedKeyWords = ["/start", "начать", "сдаюсь"];
 
-bot.on("message", msg => {
-  const chatId = msg.chat.id;
+bot.on("polling_error", m => console.log(m));
 
-  if (msg.text.toLowerCase() == "/start" && !gameStart) {
-    console.log("20 " + chatId + " : " + msg.text);
+bot.onText(/\/start/i, msg => {
```

Comment:
Here you can reuse that commands object like 
```suggestion
bot.onText(commands.START, msg => {
```

Reply:
```suggestion
bot.onText(commands.START, msg => {
```
fixed